src/org/openlcb/hub/Hub.java

```diff
@@ -101,7 +111,7 @@ public void run() {
     }
 
     BlockingQueue<Memo> queue = new LinkedBlockingQueue<>();
-    ArrayList<Forwarding> threads = new ArrayList<>();
+    List<Forwarding> threads = Collections.synchronizedList(new ArrayList<>());
```

Comment:
Since we are using this with synchronized, let's make this variable final.

Reply:
Done.  Thanks.